PdCore/src/main/java/org/puredata/android/service/PdService.java

```diff
@@ -241,8 +241,15 @@ private Notification makeNotification(Intent intent, int icon, String title, Str
 				notificationManager.createNotificationChannel(channel);
 			}
 		}
-
-		PendingIntent pi = PendingIntent.getActivity(getApplicationContext(), 0, intent, 0);
+		
+            	PendingIntent pi = null;
+            	if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
+                	pi = PendingIntent.getActivity(getApplicationContext(), 0, intent, PendingIntent.FLAG_UPDATE_CURRENT | PendingIntent.FLAG_IMMUTABLE);
+            	}
+            	else {
```

Comment:
Please apply the following formatting change to align with the rest of the code base:
```suggestion
            	} else {
```